Formula/cpmtools.rb

```diff
@@ -11,8 +11,17 @@ class Cpmtools < Formula
     sha256 "b810122c220af6b36ab9316deec811adca68313d4371f8a0121239c40b94a015" => :mavericks
   end
 
+  option "with-libdsk", "Enable libdsk support"
```

Comment:
Can remove this comment; they are autogenerated and this doensn't describe any more than the autogenerated one does. Thanks!